File: Ejercicio22_LogMaven/src/main/java/datos/Usuario.java
package datos;

public class Usuario {

	private String username;
	private String password;
	private boolean validado;
	
	public Usuario(String username, String password) {
		this.username = username;
		this.password = password;
		this.validado = false;
	}
	
	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}
	
	public boolean isValidado() {
		return validado;
	}

	public void setValidado(boolean validado) {
		this.validado = validado;
	}
}

File: Ejercicio23_LogAngular/src/app/datos/Usuario.ts
export class Usuario {

	private _username: String;
	private _password: String;

	constructor(_username: String, _password: String) {
		this._username = _username;
		this._password = _password;
	}
	
	public get username(): String {
        return this._username;
    }

    public set username(username: String) {
        this._username = username;
    }

    public get password(): String {
        return this._password;
    }

    public set password(password: String) {
        this._password = password;
    }
}
